sds_data_manager/lambda_code/SDSCode/database/models.py

```diff
@@ -1,7 +1,56 @@
 """Main file to store schema definition"""
-from sqlalchemy import Boolean, Column, DateTime, Identity, Integer, String
+
+from sqlalchemy import (
+    Boolean,
+    Column,
+    DateTime,
+    Enum,
+    ForeignKey,
+    Identity,
+    Integer,
+    String,
+)
 from sqlalchemy.orm import DeclarativeBase
 
+# Instrument name Enums for the file catalog table
+instruments = Enum(
+    "codice",
+    "glows",
+    "hi-45",
+    "hi-90",
+    "hit",
+    "idex",
+    "lo",
+    "mag",
+    "swapi",
+    "swe",
+    "ultra-45",
+    "ultra-90",
+    name="instrument",
+)
+
+# data level enums for the file catalog table
+data_levels = Enum(
+    "l0",
+    "l1a",
+    "l1b",
+    "l1c",
+    "l1ca",
+    "l1cb",
```

Comment:
where do the "a" and "b" qualifiers come from? The "pre" is a bit non-standard, but was sort of agreed upon to be included for MAG's sake. Did we have the same agreement for ca and cb?

Reply:
I wasn't there for that part of the discussion, but I got those from here:https://lasp.colorado.edu/galaxy/display/IMAP/SDC+Database+Tables in the DATA_LEVELS list

Reply:
And I got those from the diagrams here: https://lasp.colorado.edu/galaxy/display/IMAP/Instrument+Science+Data+Processing+Pipelines+Diagrams

Reply:
@tech3371 are these accurate with the a/b or did we remove those?

But, lets not let this slow us down, we can update the fields later if need be.